test/integration/deep/appprotocol/appprotocol_test.go

```diff
@@ -199,6 +201,7 @@ func runTests(ctx context.Context, t *testing.T, ns string, tcs []testCase) {
 	t.Helper()
 	for _, tc := range tcs {
 		t.Run(tc.name, func(t *testing.T) {
+			t.Helper()
```

Comment:
Not sure if it's idiomatic to add this to a `t.Run` closure. It makes sense for helper functions reused by multiple tests where we wouldn't see the calling site causing the failure, but here we wouldn't have that ambiguity.

Reply:
Yeah, I'm not to sure what to do here... i think it's probably not idiomatic to put t.Run in a helper like runTests.. it's pretty useuless to get assertion errors in this function as it is. The line number reported is useless.